<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TED Call Logger</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/icon.ico</normaloff>icon/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget_Calls">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Source Address</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Destination Address</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Destination Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Count</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Attach"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Filter">
    <property name="title">
     <string>F&amp;ilter</string>
    </property>
    <addaction name="action_Disable_all_shown"/>
    <addaction name="action_Disable_custom"/>
    <addaction name="separator"/>
    <addaction name="action_Enable_all_shown"/>
    <addaction name="action_Enable_custom"/>
    <addaction name="separator"/>
    <addaction name="action_Clear"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="action_Return_context"/>
    <addaction name="action_Return_call_stack"/>
    <addaction name="action_Return_symbols"/>
    <addaction name="action_Unsafe_memory_mode"/>
    <addaction name="action_Use_invasive_breakpoints"/>
    <addaction name="action_Auto_disable_incoming"/>
    <addaction name="separator"/>
    <addaction name="action_Set_symbol_path"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_Debug">
    <property name="title">
     <string>&amp;Debug</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Get_modules"/>
    <addaction name="action_Disassemble_address"/>
    <addaction name="action_Memory"/>
    <addaction name="separator"/>
    <addaction name="action_Enable_console"/>
    <addaction name="action_Enable_logging"/>
    <addaction name="separator"/>
    <addaction name="action_Invoke_test_function"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Options"/>
   <addaction name="menu_Filter"/>
   <addaction name="menu_Debug"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_Disable_custom">
   <property name="text">
    <string>Disable custom</string>
   </property>
  </action>
  <action name="action_Disable_all_shown">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Disable all shown</string>
   </property>
  </action>
  <action name="action_Attach">
   <property name="text">
    <string>Attach</string>
   </property>
  </action>
  <action name="action_Return_context">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Return context</string>
   </property>
  </action>
  <action name="action_Return_call_stack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Return call stack</string>
   </property>
  </action>
  <action name="action_Return_symbols">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Return symbols</string>
   </property>
  </action>
  <action name="action_Set_symbol_path">
   <property name="text">
    <string>Set symbol path</string>
   </property>
  </action>
  <action name="action_Get_modules">
   <property name="text">
    <string>Modules</string>
   </property>
  </action>
  <action name="action_Disassemble_address">
   <property name="text">
    <string>Disassemble address</string>
   </property>
  </action>
  <action name="action_Invoke_test_function">
   <property name="text">
    <string>Invoke test function</string>
   </property>
  </action>
  <action name="action_Enable_console">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable console</string>
   </property>
  </action>
  <action name="action_Enable_logging">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable logging</string>
   </property>
  </action>
  <action name="action_Memory">
   <property name="text">
    <string>Memory</string>
   </property>
  </action>
  <action name="action_Use_invasive_breakpoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use invasive breakpoints (faster)</string>
   </property>
  </action>
  <action name="action_Enable_all_shown">
   <property name="text">
    <string>Enable all shown</string>
   </property>
  </action>
  <action name="action_Enable_custom">
   <property name="text">
    <string>Enable custom</string>
   </property>
  </action>
  <action name="action_Clear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="action_Unsafe_memory_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Unsafe memory mode</string>
   </property>
  </action>
  <action name="action_Auto_disable_incoming">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto-disable incoming</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
